src/NodaTime.Web/Startup.cs

```diff
@@ -96,7 +96,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 .AddRedirect($@"^{anyVersion}/userguide$", "$1/userguide/")
                 .AddRedirect($@"^developer$", "developer/")
                 // Make /api and /userguide links to the latest stable release.
-                .AddRedirect("^(api|userguide)((?:/.*))$", "1.3.x/$1$2");
```

Comment:
Am I right in thinking that this was doing a non-greedy match for `/.*` but then adding a `$` anchor after that match?

(and so the non-greedy part is uninteresting, which is why you've removed it?)